opencog/atoms/base/Atom.h

```diff
@@ -248,6 +248,17 @@ class Atom
         setAttentionValue(new_av);
     }
 
+    /** Change the Very-Long-Term Importance */
+    void setVLTI(AttentionValue::vlti_t vltiValue)
+    {
```

Comment:
My read of the documentation said that `vlti` is a flag that means don't delete for any positive values. So it acts like a 'permanent atom' flag. However, I didn't check the EVAN code to verify that is what it does.

There is an increment and decrement that I implemented as well, and I'd expect that to be the normal interface to `vlti` for setting the flag.

The actual behavior of the existing Python interfaces was inconsistent and hard to describe. So in the spirit of simplifying and making it easier for new people to understand I changed some inconsistencies.

A particularly confusing example I encountered when testing is setting with a dictionary like:
```
>>>atom.av = {'st': 10, 'lti' 5, 'vlti': 5}
>>>atom.av
{'sti': 10, 'lti': 5, 'vlti': True}
```
Internally, the code for setting from a dictionary called increment if the value for `vlti` was `> 0` it would do an increment and decrement for `< 0`. Then the code for retrieval was treating the value as if it was a boolean and ignoring all other values. So there was no way to find out what the actual value was. And you'd get wonderful behavior like:
```
>>>atom.av = {'st': 10, 'lti' 5, 'vlti': True}
>>>atom.av = {'st': 10, 'lti' 5, 'vlti': True}
>>>atom.av = {'st': 10, 'lti' 5, 'vlti': False}
>>>atom.av
{'sti': 10, 'lti': 5, 'vlti': True}
```
which is really confusing and looks like a bug.

It seems like you'd want to be able to restore prior states which is why I changed the new implementation to make it easier to understand what is going on and make it possible to restore states.  So now you can do:
```
>>> atom.av
{'sti': 10, 'lti': 5, 'vlti': 2}
>>>original_av = atom.av
>>>
>>>... do some work ...
>>>
>>>atom.av = original_av
>>>atom.av
{'sti': 10, 'lti': 5, 'vlti': 2}
```
and
```
>>>atom.av = {'st': 10, 'lti' 5, 'vlti': 1}
>>>atom.av = {'st': 10, 'lti' 5, 'vlti': 1}
>>>atom.av = {'st': 10, 'lti' 5, 'vlti': 0}
>>>atom.av
{'sti': 10, 'lti': 5, 'vlti': 0}
```
Also, when debugging the code if you want to know why an atom is not being deleted, then it would help to know that the `vlti` value is actually `3` rather than just `True`.

So if we want to only support increment and decrement then we ought to disallow the setting of `av` with dictionaries, I think, as this usage implies a set of the value not an increment or decrement, and the previous behavior is non-intuitive.